kbase-extension/static/kbase/js/widgets/narrative_core/kbaseNarrativeSideImportTab.js

```diff
@@ -170,7 +170,7 @@ define([
 
             // Build widget container panel that holds importer inputs
             this.widgetPanel = $('<div>');
-            this.widgetPanelCard1 = $('<div style="margin: 30px 30px 0px 30px;">');
+            this.widgetPanelCard1 = $('<div style="margin: 30px 30px 0 30px;">');
```

Comment:
This file is all eslint fixes -- I came across it whilst doing a search and decided to fix the errors.